src/Universum/Catstraints.hs

```diff
@@ -0,0 +1,79 @@
+{-# LANGUAGE ConstraintKinds  #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE QuasiQuotes      #-}
+{-# LANGUAGE Rank2Types       #-}
+{-# LANGUAGE TypeApplications #-}
+{-# LANGUAGE TypeOperators    #-}
+
+-- | Provides means to envade your system with kots.
+
+module Universum.Catstraints where
+
+import           Control.Concurrent (threadDelay)
```

Comment:
These imports are not formatted according to our style-guide and latest `.stylish-haskell.yaml`.